archeryutils/classifications/classification_utils.py

```diff
@@ -267,11 +272,11 @@ def get_age_gender_step(
     """
     # There is a danger that gender step overtakes age step at U15/U16
     # interface. If this happens set to age step to align U16 with U16
-    if gender.lower() == "female" and age_cat == 3 and age_step < gender_step:
+    if gender.lower() == "female" and age_cat == 3 and age_step < gender_step:  # noqa: PLR2004 Magic Value
```

Comment:
This is a bit of a warning that passing the age category as an integer defined by steps below senior is fishy, but it works for now.

Reply:
Still using the integer, but perhaps slightly clearer in https://github.com/jatkinson1000/archeryutils/pull/81/commits/f54f4695a2096689738f2cae7d7dda5bcd1bf4c5?